exporter.php

```diff
@@ -331,7 +332,7 @@ function p4_px_single_post_filter_postmeta( $return_me, $meta_key ) {
 			global $wp_query;
 			$wp_query->in_the_loop = true;
 			for ( $i = 0; $i < count( $post_id ); $i ++ ) {
```

Comment:
Not sure if phpcs also flagged this, but here there is also a `count()` in the loop condition. You could extract it, but really the loop doesn't need to be a counter as `$i` is only used to access the current element, so it would be cleaner to instead just use `foreach( $post_ids  as $post_id ) { $next_posts = $post_id; ... }`

Reply:
Also the naming of `$post_id` is wrong and should be `$post_ids`. But I see there is a naming conflict with another variable in this file and changing it might cascade into more refactors, so you can decide if you want to take the time to fix that.

Reply:
By now I understand why you disabled phpcs for this file, as it's probably a lot of work to appropriately fix everything.

Reply:
Fixing all phpcs issues in this file are very prone to cause to break the export xml functionality. I check the top level issues and fixed them. 
Or 
We will make a ticket to refactor this functionality. (I personally prefer a separate task)